docs/source/notes/autograd.rst

```diff
@@ -245,49 +261,42 @@ where
             \frac{\partial u(x, y)}{\partial x} & \frac{\partial u(x, y)}{\partial y}\\
             \frac{\partial v(x, y)}{\partial x} & \frac{\partial v(x, y)}{\partial y} \end{bmatrix} \\
 
-This is similar to the definition of the JVP for a function defined from :math:`R^2 → R^2`, and the multiplication
-with :math:`[1, 1j]^T` is used to identify the result as a complex number.
-
-We define the :math:`VJP` of :math:`F` at :math:`(x, y)` for a cotangent vector :math:`c+dj \in C` as:
+The key assumption made in the derivation of the above formula is that function :math:`F` is a part of a
+real valued function. For more details, please check out Section 3.4 `here <https://arxiv.org/pdf/1701.00392.pdf>`_.
```

Comment:
I would maybe add another sentence here before the link to be a bit more precise like:
"It implies that the Conjugate Wirtinger and Wirtinger derivative flowing back are complex conjugate of each other. This allows us to use `grad_output` as the Conjugate Wirtinger derivative and recover the Wirtinger derivative by complex conjugation."

Reply:
gotcha! I made an update based on your suggestion. I specified that here we are talking about the incoming `Conjugate Wirtinger derivative` and `Wirtinger derivative` to ensure the reader doesn't confuse them with the Wirtinger derivatives for `F`